--- gmail-0.6.0/configure.in~	Fri Sep 12 00:40:55 2003
+++ gmail-0.6.0/configure.in	Fri Sep 12 00:40:55 2003
@@ -22,15 +22,15 @@
 [  --with-mysql=PFX        Prefix where MySQL is installed (usually /usr/local)])
 
 if test "x$with_mysql" != "x"; then
-  AC_CHECK_LIB_LOC($with_mysql/lib/mysql,mysqlclient,mysql_connect,,
+  AC_CHECK_LIB_LOC($with_mysql/lib/mysql,mysqlclient,mysql_init,,
     AC_MSG_ERROR(Need MySQL client library!))
 else
-  AC_CHECK_LIB_LOC(/opt/lib/mysql,mysqlclient,mysql_connect,,
-    AC_CHECK_LIB_LOC(/usr/local/lib/mysql,mysqlclient,mysql_connect,,
-    AC_CHECK_LIB_LOC(/usr/lib/mysql,mysqlclient,mysql_connect,,
-    AC_CHECK_LIB_LOC(/usr/local/lib,mysqlcient,mysql_connect,,
-    AC_CHECK_LIB_LOC(/opt/lib,mysqlclient,mysql_connect,,
-    AC_CHECK_LIB_LOC(/usr/lib,mysqlclient,mysql_connect,,
+  AC_CHECK_LIB_LOC(/opt/lib/mysql,mysqlclient,mysql_init,,
+    AC_CHECK_LIB_LOC(/usr/local/lib/mysql,mysqlclient,mysql_init,,
+    AC_CHECK_LIB_LOC(/usr/lib/mysql,mysqlclient,mysql_init,,
+    AC_CHECK_LIB_LOC(/usr/local/lib,mysqlclient,mysql_init,,
+    AC_CHECK_LIB_LOC(/opt/lib,mysqlclient,mysql_init,,
+    AC_CHECK_LIB_LOC(/usr/lib,mysqlclient,mysql_init,,
     AC_MSG_ERROR(Need MySQL client library!)))))))
 fi
 
--- gmail-0.6.0/configure~	Fri Sep 12 00:40:55 2003
+++ gmail-0.6.0/configure	Fri Sep 12 00:40:55 2003
@@ -2544,10 +2544,10 @@
 #include "confdefs.h"
 /* We use char because int might match the return type of a gcc2
     builtin and then its argument prototype would still apply.  */
-char mysql_connect();
+char mysql_init();
 
 int main() {
-mysql_connect()
+mysql_init()
 ; return 0; }
 EOF
 if { (eval echo configure:2554: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
@@ -2586,10 +2586,10 @@
 #include "confdefs.h"
 /* We use char because int might match the return type of a gcc2
     builtin and then its argument prototype would still apply.  */
-char mysql_connect();
+char mysql_init();
 
 int main() {
-mysql_connect()
+mysql_init()
 ; return 0; }
 EOF
 if { (eval echo configure:2596: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
@@ -2624,10 +2624,10 @@
 #include "confdefs.h"
 /* We use char because int might match the return type of a gcc2
     builtin and then its argument prototype would still apply.  */
-char mysql_connect();
+char mysql_init();
 
 int main() {
-mysql_connect()
+mysql_init()
 ; return 0; }
 EOF
 if { (eval echo configure:2634: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
@@ -2662,10 +2662,10 @@
 #include "confdefs.h"
 /* We use char because int might match the return type of a gcc2
     builtin and then its argument prototype would still apply.  */
-char mysql_connect();
+char mysql_init();
 
 int main() {
-mysql_connect()
+mysql_init()
 ; return 0; }
 EOF
 if { (eval echo configure:2672: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
@@ -2687,23 +2687,23 @@
 
 else
   echo "$ac_t""no" 1>&6
-echo $ac_n "checking for libmysqlcient in /usr/local/lib""... $ac_c" 1>&6
-echo "configure:2692: checking for libmysqlcient in /usr/local/lib" >&5
-ac_lib_var=`echo /usr/local/lib'_'mysqlcient | sed 'y%./+-%__p_%'`
+echo $ac_n "checking for libmysqlclient in /usr/local/lib""... $ac_c" 1>&6
+echo "configure:2692: checking for libmysqlclient in /usr/local/lib" >&5
+ac_lib_var=`echo /usr/local/lib'_'mysqlclient | sed 'y%./+-%__p_%'`
 if eval "test \"`echo '$''{'ac_cv_lib_loc_$ac_lib_var'+set}'`\" = set"; then
   echo $ac_n "(cached) $ac_c" 1>&6
 else
   ac_save_LIBS="$LIBS"
-LIBS="-L/usr/local/lib -lmysqlcient  $LIBS"
+LIBS="-L/usr/local/lib -lmysqlclient  $LIBS"
 cat > conftest.$ac_ext <<EOF
 #line 2700 "configure"
 #include "confdefs.h"
 /* We use char because int might match the return type of a gcc2
     builtin and then its argument prototype would still apply.  */
-char mysql_connect();
+char mysql_init();
 
 int main() {
-mysql_connect()
+mysql_init()
 ; return 0; }
 EOF
 if { (eval echo configure:2710: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
@@ -2721,7 +2721,7 @@
 fi
 if eval "test \"`echo '$ac_cv_lib_loc_'$ac_lib_var`\" = yes"; then
   echo "$ac_t""yes" 1>&6
-  LIBS="-L/usr/local/lib -lmysqlcient $LIBS"
+  LIBS="-L/usr/local/lib -lmysqlclient $LIBS"
 
 else
   echo "$ac_t""no" 1>&6
@@ -2738,10 +2738,10 @@
 #include "confdefs.h"
 /* We use char because int might match the return type of a gcc2
     builtin and then its argument prototype would still apply.  */
-char mysql_connect();
+char mysql_init();
 
 int main() {
-mysql_connect()
+mysql_init()
 ; return 0; }
 EOF
 if { (eval echo configure:2748: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
@@ -2776,10 +2776,10 @@
 #include "confdefs.h"
 /* We use char because int might match the return type of a gcc2
     builtin and then its argument prototype would still apply.  */
-char mysql_connect();
+char mysql_init();
 
 int main() {
-mysql_connect()
+mysql_init()
 ; return 0; }
 EOF
 if { (eval echo configure:2786: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
